source/src/mesh/cs_MeshMsgHandler.cpp

```diff
@@ -650,24 +634,24 @@ void MeshMsgHandler::replyWithRetCode(cs_mesh_model_msg_type_t type, cs_ret_code
 	if (reply->buf.len < sizeof(cs_mesh_model_msg_result_header_t)) {
 		return;
 	}
+	// TODO: we could save a byte by sending the same mesh message type (with a different payload) instead of a result mesh message.
```

Comment:
```suggestion
	// Optimization suggestion: we could save a byte by sending the same mesh message type (with a different payload) instead of a result mesh message.
```